API/SSW.Consulting/Program.cs

```diff
@@ -34,6 +35,7 @@ public static async Task Main(string[] args)
                 {
                     var logger = scope.ServiceProvider.GetRequiredService<ILogger<Program>>();
                     logger.LogError(ex, "An error occurred while migrating or initializing the database.");
+                    return;
```

Comment:
Returning here makes sense if migrations failed.

Thoughts?

Reply:
do we need to flush the log before returning?